ui/components/src/Legend/ListLegend.tsx

```diff
@@ -46,9 +46,7 @@ export function ListLegend({ items, height, width, selectedItems, onLegendItemCl
             isVisuallySelected={isLegendItemVisuallySelected(item, selectedItems)}
             onClick={onLegendItemClick}
             sx={{
-              // Having an explicit width is important for the ellipsizing to
-              // work correctly. Subtract padding to simulate padding.
-              width: width,
```

Comment:
Note: this still works fine. You can see the ellipsizing working properly in http://localhost:6006/?path=/story/components-legend--right-with-long-labels